documentation/asciidoc/topics/proc_installing_client_nix.adoc

```diff
@@ -25,10 +25,10 @@ ifdef::downstream[]
 | Repository
 
 | {rhel_short} 7
-| `jb-datagrid-8.1-for-rhel-7-server-rpms`
+| `jb-datagrid-{ProductVersion}-for-rhel-7-server-rpms`
```

Comment:
Hey @oraNod  the clients are going to ship in the `8.1` repo, since there is not an `8.2` repo

Reply:
thanks @alanfx I've reverted that back to 8.1.